README.md

```diff
@@ -1,233 +1,249 @@
-# Publishing Information
-
-## Purpose
-
-The Publishing & Information Hub will be responsible for the following:
-
--   Receiving data from source systems, such as Schedule & Listing, via Hearings Management Interface
--   Publish lists, outcomes, judgements and management information onto GOV.UK
--   Provide functionality to display information in court and tribunals buildings on the relevant hardware
--   Comply with Open Justice procedures and business rules
+# pip-frontend
+![Light Mode Logo](lightLogo.png#gh-light-mode-only)
+![Dark Mode Logo](darkLogo.jpg#gh-dark-mode-only)
```

Comment:
The logo is showing weirdly on my screen. Can see the outline of the light mode logo and also the dark mode logo. Not sure if we need the logo at all.

![image](https://user-images.githubusercontent.com/80688761/229771284-dd2cfc2a-715c-4160-979c-bea3897c1a4a.png)

Reply:
I had the same issue, however it only seems to happen locally on the machine. In github all seems OK. Could you check in github to see if it's the same for you?